### ISSUE TITLE
Add example project.

### ISSUE THREAD

**Issue by mike866** [TrueCar/react-launch-darkly#16]
We should include example project. Just a simple project to show implementation details.

**Comment by sethbattin** [comment]
I agree.  Using this component is complicated by needing an account key in order for the external service to work correctly.  But I would expect that anyone attempting to use the react component is willing to also use the service itself.

There's an example usage here:  https://github.com/TrueCar/react-launch-darkly/pull/6.  The config file setting `launchDarkly.clientId` is what would need to be replaced with a user's key.  The example also references `Helmet`, which could be removed.
